satellite/orders/service.go

```diff
@@ -23,6 +24,12 @@ import (
 	"storj.io/storj/pkg/storj"
 )
 
+// TODO: if/v3-1927 If this value should be set through settings, where should
+// it be?
+// The used value must be changed when the data science team provide a value
+// based on an analysis of the network behavior.
+var optimalThresholdExceessRate = 0.05 // 5%
```

Comment:
Move this value to be a config in https://godoc.org/storj.io/storj/pkg/datarepair/repairer#Config